arctic/serialization/numpy_strings.py

```diff
@@ -0,0 +1,179 @@
+import logging
+import numpy as np
+import numpy.ma as ma
+import pandas as pd
+
+from bson import Binary, SON
+
+from .._compress import compress, decompress
+
+
+class NumpyArrayConverter(object):
+    """
+    Converts a Numpy ndarray to and from PyMongo SON representation:
+
+        {
```

Comment:
This looks so similar to the tickstore column schema (addition of a length, and removal of a bitmask):
https://github.com/manahl/arctic/blob/master/arctic/tickstore/tickstore.py#L37

Can it be made compatible?